.circleci/config.yml

```diff
@@ -1,104 +1,148 @@
-version: 2
+version: 2.1
+
+aliases:
+  # Define paths and never think about them again
+  - &GAIA /tmp/gaia
+  - &WORKSPACE /tmp/voyager
+  - &DIST /tmp/voyager/app/dist
+  # Pick docker versions here only, then use the aliases in the executors definition
+  - &docker-node circleci/node:10.15
+  - &docker-browsers circleci/node:10.15-browsers
+  - &docker-go circleci/golang:1.11
+
+commands:
+  yarn-update:
+    description: "[YARN] fetch new version"
+    steps:
+      - run:
+          name: Update Yarn
+          command: sudo npm -g uninstall yarn ; sudo npm -g install yarn@1.13.0
```

Comment:
why did you fix the version to 1.13.0?

Reply:
The security is available only since 1.13.0 (fixed) they delivered it in 1.12.x (where x was a bit bigger than our version) so once I had to update I decided to apply the fixed version

Reply:
But the security feature is still there in a later version so I was asking why we need to fix to that specific version? we might loose security updates this way.

Reply:
Ah got it! mostly the same reason why we are setting the node version in the docker files, to be sure they don't insert some change we haven't tested.

Initially I tried with @latest, should we go that way? yarn seems stable in his future releases, no breaking changes until 2.0, and even there seems ok for our needs

Reply:
is this question resolved?

Reply:
Ok good argument. I would anyway go with at least allowing patch versions, so `1.13.x`. But I think we would be fine with `1.x`.

Reply:
Ok, now this logic will be moved away from here, it has been set into tendermint/devops repo